_posts/2024-01-10-15.0.0-release.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: "Apache Arrow 15.0.0 Release"
+date: "2024-01-10 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 15.0.0 release. This covers
+over 3 months of development work and includes [**XXX resolved issues**][1]
+from [**YYY distinct contributors**][2]. See the [Install Page](https://arrow.apache.org/install/)
+to learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 14.0.0 release, Curt Hagenlocher, Xuwei Fu, James Duong and Felipe Oliveira Carvalho
+have been invited to be committers.
+Jonathan Keane and Raúl Cumplido have joined the Project Management Committee (PMC).
+
+As per our tradition of rotating the PMC chair once a year
+Andy Grove was elected as the new PMC chair and VP.
+
+Thanks for your contributions and participation in the project!
+
+## Columnar Format Notes
+
+## C Data Interface notes
+
+## Arrow Flight RPC notes
+
+## C++ notes
+
+### Compute layer
+
+### Datasets
+
+### Gandiva
+
+### Parquet
+
+### Substrait
+
+### Miscellaneous
+
+## C# notes
+
+## Go Notes
+
+### Bug Fixes
+
+### Enhancements
+
+## Java notes
+
+## JavaScript notes
```

Comment:
@domoritz @trxcllnt could you help with the notes for the release blog post?

Reply:
```suggestion
## JavaScript notes

* [GH-39017: [JS] Add typeId as attribute](https://github.com/apache/arrow/pull/39018)
* [GH-39257: [JS] LargeBinary](https://github.com/apache/arrow/pull/39258)
* [GH-15060: [JS] Add LargeUtf8 type](https://github.com/apache/arrow/pull/35780)
* [GH-39259: [JS] Remove getByteLength](https://github.com/apache/arrow/pull/39260) 
* [GH-39435: [JS] Add Vector.nullable](https://github.com/apache/arrow/pull/39436)
* [GH-39255: [JS] Allow customization of schema when passing vectors to table constructor](https://github.com/apache/arrow/pull/39256)
* [GH-37983: [JS] Allow nullable fields in table when constructed from vector with nulls](https://github.com/apache/arrow/pull/39254)
* [GH-39251: [JS] Use resizable buffer in builder](https://github.com/apache/arrow/pull/39252)
* [GH-39289: [JS] Add types to exports](https://github.com/apache/arrow/pull/39475)
```

Reply:
@domoritz should I remove:
```
* [GH-39251: [JS] Use resizable buffer in builder](https://github.com/apache/arrow/pull/39252)
 ```
from the blog post?

Reply:
Done